draft-sharabayko-srt.md

```diff
@@ -3199,21 +3208,24 @@ all SRT users speak the same language. The intent of the convention is to:
 ## General Syntax
 
 This recommended syntax starts with the characters known as an executable specification in POSIX: `#!`.
-The first following character is `:` that marks the YAML format.
-As the content format, key-value pair or nesting block can be presented.
+
+The next character defines the format used for the following key-value pair syntax.
+At the moment, there is only one supported syntax identified by `:` and described below.
+
+Everything that comes after syntax identifier is further referenced as the content of Stream ID.
+
+The content starts with `:` or `{` character identifying the its format:
 
 `:`
-: the comma-separated keys with no nesting
+: a comma-separated key-value pairs with no nesting,
```

Comment:
```suggestion
: a comma-separated key-value pair with no nesting,
```

Reply:
@cmollahan Perhaps,

> comma-separated key-value pairs with no nesting,

because it can be one or several pairs in the following format
`key1=value1,key2=value2,...`?

Reply:
Yes, that sounds good. I would just remove the 'a' at the beginning of the line.